apps/web/src/styles/theme.ts

```diff
@@ -0,0 +1,167 @@
+  "3xl": {
+    fontSize: "30px",
+  },
+  "4xl": {
+    fontSize: "36px",
+  },
+  "5xl": {
+    fontSize: "48px",
+  },
+  "6xl": {
+    fontSize: "69px",
+  },
+};
+
+const theme = extendTheme({
+  components: {
+    Text: {
+      sizes,
+    },
+    Heading: {
+      baseStyle: {
+        fontWeight: "600",
+        fontFamily: "Inter",
+      },
+      sizes,
+    },
+    Card: {
+      baseStyle: {
+        container: {
+          color: "inherit",
+          bg: "var(--chakra-colors-white)",
+          boxShadow: "2px 4px 12px 0px rgba(45, 55, 72, 0.05)",
+        },
+      },
+    },
+    Input: (props: StyleFunctionProps) => ({
+      sizes: { md: { field: { height: "48px" } } },
+      defaultProps: { variant: "filled" },
+      variants: {
+        filled: {
+          field: {
+            bg: mode(light.grey[800], dark.grey[800])(props),
+            border: "1px solid",
+            borderColor: mode(light.grey[500], dark.grey[500])(props),
+            _invalid: {
+              borderColor: light.red,
+            },
+            _hover: {
+              bg: mode(light.grey[800], dark.grey[800])(props),
+            },
+            _focusVisible: {
+              bg: mode(light.grey[800], dark.grey[800])(props),
+              borderColor: mode(light.grey[400], dark.grey[400])(props),
+            },
+          },
+        },
+      },
+    }),
+    FormLabel: {
+      baseStyle: {
+        fontWeight: "600",
+        marginBottom: "12px",
+        size: "md",
+      },
+      sizes,
+    },
+    Tabs: {
+      baseStyle: {
+        tab: {
+          color: "green",
+          _selected: {
+            bg: "var(--chakra-colors-gray-200)",
+            color: "var(--chakra-colors-gray-900)",
+          },
+        },
+      },
+    },
+    Switch: {
+      baseStyle: (props: StyleFunctionProps) => ({
+        track: {
+          backgroundColor: mode(light.grey[300], dark.grey[300])(props),
+          _checked: {
+            backgroundColor: mode(light.grey[400], dark.grey[400])(props),
+          },
+        },
+        thumb: {
+          backgroundColor: mode(light.grey.white, dark.grey.white)(props),
+        },
+      }),
+    },
+    Link: {
+      baseStyle: (props: StyleFunctionProps) => ({
+        color: mode(light.grey[600], dark.grey[600])(props),
+      }),
+    },
+    Button: {
+      variants: {
+        solid: {
+          bg: "var(--chakra-colors-gray-100)",
+          _dark: {
+            _hover: {
+              bg: "var(--chakra-colors-gray-300)",
+            },
+          },
+        },
+      },
+    },
+  },
+  config,
+  styles: {
+    global: (props: StyleFunctionProps) => ({
+      body: {
+        color: mode(light.grey[600], dark.grey[600])(props),
+        bgColor: mode(light.bg, dark.bg)(props),
+        bgImage: mode(light.bgGradient, dark.bgGradient)(props),
+        _before: {
+          bgColor: mode(light.bgMaskColor, dark.bgMaskColor)(props),
+          maskImage: "url(src/assets/bg.svg)",
```

Comment:
this works only for the light bg, right?

Reply:
it works either